src/context.rs

```diff
@@ -127,64 +131,64 @@ impl Context {
     #[allow(clippy::too_many_arguments)]
     pub fn start_auth_session(
         &mut self,
-        tpm_key: ESYS_TR,
+        tpm_key: Option<KeyHandle>,
         bind: ESYS_TR,
```

Comment:
Could this also be replaced with `Option<ObjectHandle>`?

Reply:
Probably I just need to do some digging in the specification to see what it should be...

Reply:
> The  entity  referenced  with  the bind parameter  contributes  an  authorization  value  to  the sessionKeygeneration process.
>
> entity providing the authValue; may be TPM_RH_NULL

I think it can be any entity that sports a handle

Reply:
Yeah seams as you could send in anything that handle that refers to something that some kind of auth but the question is should None => ESYS_TR_NONE or ESYS_TR_RH_NULL?

Reply:
Nevermind it should be none. Yeah, lets do that.